common/src/main/java/com/datastrato/gravitino/dto/CatalogDTO.java

```diff
@@ -234,6 +235,11 @@ public CatalogDTO build() {
           StringUtils.isNotBlank(provider), "provider cannot be null or empty");
       Preconditions.checkArgument(audit != null, "audit cannot be null");
 
+      // redact properties
+      if (properties != null) {
```

Comment:
@jerryshao 
I'm not very sure about the point:  Should we only need to apply the action for catalog, what's about `schema` and `table`, possibly there would be sensitive messages in the schema/table properties.

Reply:
The most common encryption scenario at present is the connection information for the catalog (mainly password). As for schemas/tables, there is no need for this in our production enviorment